drivers/source/QSPI.cpp

```diff
@@ -255,6 +281,24 @@ bool QSPI::_initialize()
     return _initialized;
 }
 
+// Note: Private helper function to initialize qspi HAL
+bool QSPI::_initialize_direct()
```

Comment:
`_acquire` is still just calling `_initialize` - don't you need to do the `_init_func` thing that SPI had?

Reply:
`QSPI` constructor calls HAL `init()` function, but I missed that `aquire()` function also calls init(). You are right that the same mechanism as in SPI must be also used here.